docs/classes/ChatGPTAPIBrowser.md

```diff
@@ -48,6 +48,7 @@ Creates a new client for automating the ChatGPT webapp.
 | `opts.email` | `string` | - |
 | `opts.executablePath?` | `string` | **`Default Value`**  `undefined` * |
 | `opts.isGoogleLogin?` | `boolean` | **`Default Value`**  `false` * |
+| `opts.isWindowsLogin?` | `boolean` | **`Default Value`**  `false` * |
```

Comment:
in the future, remove all auto-generated artifacts from PRs especially if they contain git links. generally, the repo author will want to do this as a prepublish step before publishing to NPM.

this helps to keep PRs focused on actual content changes.

Reply:
It looks like this was created by hand; on my end, I run `pnpm run docs` to auto-generate `docs/` before publishing to npm, so you don't need to update by hand.